include/igl/opengl/glfw/imgui/ImGuiMenu.cpp

```diff
@@ -335,6 +335,7 @@ IGL_INLINE void ImGuiMenu::draw_labels_window()
 
 IGL_INLINE void ImGuiMenu::draw_labels(const igl::opengl::ViewerData &data)
 {
+  // Alec: How can we get these to respect (optionally) the depth of the scene?
```

Comment:
It should be possible to do with ImGui, albeit a bit difficult. See e.g. [this discussion](https://github.com/ocornut/imgui/issues/1162). There are other important issues with our current setup for drawing labels (see #1109), so we might be better off re-implementing 3D labels outside of the ImGui framework.